m4opt/_cli/schedule.py

```diff
@@ -46,7 +46,9 @@ def schedule(
             help="Output filename for generated schedule", metavar="SCHEDULE.ecsv"
         ),
     ],
-    mission: MissionOption = missions.uvex,
+    mission: Annotated[
+        str, typer.Option(help="Mission to use for scheduling", case_sensitive=False)
+    ] = "uvex",
```

Comment:
Why did you replace the original implementation of the mission option?

Reply:
The old one was pointing to uvex,  `mission: MissionOption = missions.uvex
This did not match when using "ultrasat" as an argument `--mission ultrasat`.

Reply:
That is the default value. If the processing of the command line arguments is broken, then please open a separate issue or PR about that.

Reply:
Got it, I will reopen a separate PR for each.